torchdata/dataloader2/communication/eventloop.py

```diff
@@ -109,7 +109,7 @@ def DataPipeToQueuesLoop(source_datapipe, req_queue, res_queue, process_name, ca
     source_datapipe = extract_wrapper(source_datapipe)
 
     if call_on_process_init is not None:
-        call_on_process_init(source_datapipe)
+        source_datapipe = call_on_process_init(source_datapipe)
```

Comment:
Can we add some documentation or typing for `call_on_process_init` somewhere within the eventloop.py file?

Reply:
Do you mean adding inline doc for the arguments of `DataPipeToQueuesLoop`?

Reply:
Yes, or anywhere else that makes sense

Reply:
Sounds good